Resources/Accessibility/Colors.md

```diff
@@ -3,12 +3,21 @@
 
 ## Why should you care?
 
+* Everybody uses the internet and each of them are different! Its unlikely that you will know every user so creating a website that some can't use is annoying to them and detrimental to your websites growth. Having more users helps expand the site's reach and increases the likelyhood of gaining more users in the future.
```

Comment:
Consider making the section bullet points into paragraphs, like an article blog, rather than bullet points for separate thoughts.

Small grammar fix:  Its in the second sentence should be It's

Reply:
Fixed grammar and swapped order.